Builder/Exception/MissingAssociationAdmin.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\AdminBundle\Builder\Exception;
+
+/**
+ * Signals that a field description is missing an association admin when a form type requires it.
+ *
+ * @author ju1ius <ju1ius@laposte.net>
+ */
+final class MissingAssociationAdmin extends \LogicException
```

Comment:
Off-topic, and personal taste: objects should speak the language of the domain.
`*Interface` and `*Trait` suffixes make sense, because they are not objects.
`Exception`s are objects, and PHP only allow throwing `Exception`s (or `Throwable`s).

I find it much more meaningful to have a sub-namespace for exceptions, and 
```php
throw new ThisShouldNotHappen();
// rather than
throw new ThisShouldNotHappenException();
```
Having the exception suffix encourages bad/overly-generic names, IMO.
The only case you would need suffixing is when your exception class is abstract/non-final.